components/drift-detection/cifar10/cifar10_drift.ipynb

```diff
@@ -37,9 +37,17 @@
   },
   {
    "cell_type": "code",
```

Comment:
I notice this notebook still says "Tested on GKE with Knative 0.13 and Istio 1.3.1". We should change that to 0.18 and 1.7

Reply:
Done